docs/vs-requests.md

```diff
@@ -3,8 +3,6 @@
 Although we try our best to mimic the requests API, some functionality is not easy to implement and left out.
 Here are a list of known incompatibilities:
 
-- retries are not supported yet, tracked in [#24](https://github.com/yifeikong/curl_cffi/issues/24)
-- stream/iterate are not supported. tracked in [#26](https://github.com/yifeikong/curl_cffi/issues/26)
 - files are not supported.
-- session cookies may lost during redirects, tracked in [#55](https://github.com/yifeikong/curl_cffi/issues/55)
```

Comment:
About cookies, under some discouraged condition, it's still lost. Could you please add some information about [this](https://github.com/yifeikong/curl_cffi/issues/119#issuecomment-1737724223)?

Reply:
Done, I'd recommend reopen #55 to keep tracking of that issue.